commons/utils/s3_utils.py

```diff
@@ -389,3 +390,10 @@ def assert_s3_err_msg(rgw_error, cortx_error, cmn_cfg, error):
         assert_utils.assert_in(rgw_error, error.message, error.message)
     else:
         assert_utils.assert_in(cortx_error, error.message, error.message)
+
+
+def create_random_file(file_path, min_size, max_size):
+    """Create a random sized file in the specified range in MB."""
+    with open(file_path, 'wb') as fout:
+        fout.write(os.urandom(randint(1024000 * int(min_size),  # nosec
```

Comment:
To make it more readable make use of Sizes.MB from commons/constants.py

Reply:
makes sense, updated the as per the suggestion